app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Users::OmniauthCallbacksController < ApplicationController
   # TODO: Remove this once we figure out IST, this is only required for the omniauth developer provider
-  skip_before_action :verify_authenticity_token, only: :complete unless Rails.env.production?
+  skip_before_action :verify_authenticity_token, only: :complete unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
```

Comment:
We could override the view to add the token? I dunno, it's not a deal breaker but it make me nervous anytime something is potentially disabling the auth_token checks. Thumbs down, omniauth devs. Add a soft-check for the method definition